.gitignore

```diff
@@ -99,3 +99,10 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# config/tweet files
+config.json
```

Comment:
We can add a `sample.config.json` listing all the variables needed to run this app.

Users can then simply rename this file to `config.json` with appropriate values.

Reply:
Right now the `config.json` is created by the program itself with all the appropriate fields on the first run and user is notified of the changes they have to make in order to make it work.

Reply:
Yeah, I remember that. Thanks for bringing it up again. I forgot about this in the last review pass.